### PR TITLE
Version 0.9.5

### PR DESCRIPTION
**Pull request by jimbru**
Main reason I want this is for the http changes.

### CODE REVIEW

File: src/common/http.clj [hunk 1]
```diff
@@ -42,12 +42,15 @@
         (:body response)))))
 
 (defn- process-response
-  [response]
+  [response unwrap]
```
Comment (ctdean) [inline src/common/http.clj]: How about `unwrap?`

File: src/common/http.clj [hunk 2]
```diff
@@ -9,7 +9,7 @@
   [2] https://github.com/dakrone/clj-http#headers
 
   >> ctdean jimbru"
-  (:refer-clojure :exclude [get])
+  (:refer-clojure :rename {get core-get})
```
Comment (ctdean) [inline src/common/http.clj]: fancy

**Comment by ctdean** [comment]
+1

File: src/common/http.clj (hunk 1 above)
Comment (jimbru) [inline src/common/http.clj]: Sounds good.

File: src/common/http.clj (hunk 2 above)
Comment (jimbru) [inline src/common/http.clj]: http://media.moddb.com/images/members/1/231/230182/le_fancy.jpg

**Comment by keithballinger** [comment]
I miss you guys.

**Comment by jimbru** [comment]
We miss you too @keithballinger :heart:

**Comment by venantius** [comment]
I miss you guys too!

**Comment by keithballinger** [comment]
Let's have a reunion drinks and/or dinner!

**Pull request merged**